============================================================
--- Makefile.am	ecc00e0b8e9b5350157a1922e430ade4508d31bd
+++ Makefile.am	6f03f3c51fd20e87eb1a044a7ae1772b3acd722e
@@ -405,7 +405,7 @@ endif
 
 # i18n support
 
-SUBDIRS = po
+SUBDIRS = po . mtn_cvs
 datadir = @datadir@
 localedir = $(datadir)/locale
 DEFS = -DLOCALEDIR=\"$(localedir)\" @DEFS@
============================================================
--- commands.cc	b75ca8fe6164c20993fe90c5383ebc888e2eae98
+++ commands.cc	5593222227701d6cbf8b673bfd77b5ab65b04fbb
@@ -351,6 +351,7 @@ CMD(crash, hidden_group(), "{ N | E | I 
 #undef maybe_throw_bare
 }
 
+#ifndef LIBMTN_COMPILE
 string
 describe_revision(app_state & app,
                   revision_id const & id)
@@ -506,6 +507,8 @@ process_commit_message_args(bool & given
     given = false;
 }
 
+#endif
+
 // Local Variables:
 // mode: C++
 // fill-column: 76
============================================================
--- configure.ac	514115197e68db548065f885f801415f6b7cb38f
+++ configure.ac	f03fe8d606ea1f9ea4bba3d1adfe8f1a4b7eb4d4
@@ -6,7 +6,7 @@ AM_CONFIG_HEADER([config.h])
 AM_INIT_AUTOMAKE([1.9 tar-ustar std-options])
 AC_CONFIG_SRCDIR([app_state.cc])
 AM_CONFIG_HEADER([config.h])
-AC_CONFIG_FILES([Makefile])
+AC_CONFIG_FILES([Makefile mtn_cvs/Makefile])
 
 # Checks for programs.
 dnl Note: AC_PROG_CC is implied by AM_PROG_CC_C_O and in any case has
============================================================
--- ui.cc	21bba61b39d2178a31aa97437d0b60688003ff42
+++ ui.cc	ea3134e946bf6d91320782c1965d21d43623f15b
@@ -112,6 +112,17 @@ ticker::operator++()
     ui.write_ticks();
 }
 
+void 
+ticker::operator--()
+{
+  I(ui.tickers.find(keyname) != ui.tickers.end());
+  I(ticks);
+  ticks--;
+  ui.some_tick_is_dirty = true;
+  if (ticks % mod == 0)
+    ui.write_ticks();
+}
+
 void
 ticker::operator+=(size_t t)
 {
============================================================
--- ui.hh	ea2c11aaa5de4bc00851ee8bb7a9730f509c93fc
+++ ui.hh	3ea8ae8257415bf8adf7845bb1a4b4279923941f
@@ -40,6 +40,7 @@ struct ticker
   void set_total(size_t tot) { use_total = true; total = tot; }
   void set_count_size(size_t csiz) { count_size = csiz; }
   void operator++();
+  void operator--();
   void operator+=(size_t t);
   ~ticker();
 };
